python/src/ops.cpp

```diff
@@ -3027,6 +3027,22 @@ void init_ops(nb::module_& m) {
           inclusive (bool): The i-th element of the output includes the i-th
             element of the input.
       )pbdoc");
+  m.def(
+      "conjugate",
```

Comment:
NumPy has both `np.conj` and `np.conjugate` which are aliases. Not sure if we want to do the same thing.

Reply:
I would say yes